interfaces/cython/cantera/examples/reactors/fuel_injection.py

```diff
@@ -60,30 +60,37 @@ def fuel_mdot(t):
         states.append(r.thermo.state, t=tnow)
 
 # nice names for species
-labels = {
+species_aliases = {
+    'o2': 'O$_2$',
+    'h2o': 'H$_2$O',
+    'co': 'CO',
+    'co2': 'CO$_2$',
+    'h2': 'H$_2$',
+    'ch4': 'CH$_4$'
+}
+pah_aliases = {
     'A1c2h': 'phenylacetylene',
     'A1c2h3': 'styrene',
     'A1': 'benzene',
     'A2': 'naphthalene',
     'A2r5': 'acenaphthylene',
     'A3': 'phenanthrene',
-    'A4': 'pyrene',
-    'o2': 'O$_2$',
-    'h2o': 'H$_2$O',
-    'co2': 'CO$_2$',
-    'h2': 'H$_2$',
-    'ch4': 'CH$_4$'
+    'A4': 'pyrene'
 }
+gas.case_sensitive_species_names = True
+for name, alias in {**pah_aliases, **species_aliases}.items():
+    gas.add_species_alias(name, alias)
```

Comment:
A small thing, but I think it would be both clearer and (less importantly, since it would be a very small change 😄 ) more performant to split this into two loops.

Reply:
yes, it is clearer to split ...